ibm_catalog.json

```diff
@@ -50,41 +58,67 @@
             ],
             "configuration" : [
               {
-                "key": "ibmcloud_api_key"
-              },
-              {
-                "key": "use_existing_resource_group"
+                "key": "ibmcloud_api_key",
+                "required": true,
```

Comment:
this is not needed as its a special input required by everything in projects. No need to set display the name or description it has its own special UI 
![image](https://github.com/user-attachments/assets/f69f304c-aacb-435a-be26-77f911d8c1f3)